### DISCUSSION TITLE
Ajout de familiers à la rédaction

### DISCUSSION THREAD

**Discussion by SomniumWed** [Crownicles/Crownicles#1369] [Nouvelles idées]
Hello ! 
Je trouve que ce serait pas mal de pouvoir donner un pet (aléatoire) au joueur sur des évents (entre autre pour les rédacteurs. Ce serait plus diversifié si on pouvait en plus choisir si le joueur reçoit un pet ou pas durant une issue de son évent).

**Comment by Pietagorh** [comment]
Va y avoir du travail fait sur les récompenses t'inquiète
